source/dpp/runtime/options.d

```diff
@@ -68,17 +69,42 @@ struct Options {
             args[1..$].filter!(a => a.extension != ".dpp").array ~
             dFileNames;
 
+        // use msvcrtd.lib if no other libc is specified
+        bool addNODEFAULTLIB = false;
```

Comment:
`version(Windows)`

Reply:
Fixed in https://github.com/atilaneves/dpp/pull/285/commits/23197616fb9f7ce4795855934075e6d0b1707d65